save-common/src/commonMain/kotlin/org/cqfn/save/core/plugin/PluginConfig.kt

```diff
@@ -77,6 +79,8 @@ data class GeneralConfig(
     val suiteName: String? = null,
     val excludedTests: List<String>? = null,
     val expectedWarningsPattern: Regex? = null,
+    val expectedWarningsMiddlePattern: Regex? = null,
+    val expectedWarningsEndPattern: Regex? = null,
```

Comment:
Does Kotlin/Native have multiline regexes? Or maybe specifying `expectedWarningsPattern` with explicit end can be enough? Like `\/\* ;warn[\S\s]*\*\/$`?

Reply:
Nevermind, we still need to extract warning text from every line.